### DISCUSSION TITLE
RocksDB backend

### DISCUSSION THREAD

**Discussion by kordano** [replikativ/datahike#416] [Ideas]
### Describe the feature you would like to request

It would be good to have [RocksDB](http://rocksdb.org/) as a simple backend alternative.

### Describe the solution you would like

A `datahike-rocksdb` library that implements the latest konserve protocols.

### Describe alternatives you've considered

Also good would be LevelDB.

**Comment by kordano** [comment]
What do you need to work on this @jsmassa?

**Comment by TimoKramer** [comment]
https://github.com/replikativ/konserve-rocksdb Konserve backend done? How about the datahike-backend?
